apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -48,11 +48,16 @@ let userInOptimizelyVariant = experiments.isEnabled(
 
 $(document).ready(() => {
   const schoolInfoMountPoint = document.getElementById('school-info-inputs');
+
+  const inClearerUserTypeOptExp =
+    localStorage.getItem('inClearerUserTypeOptExpLS') === 'true';
```

Comment:
One question, when is the `inClearerUserTypeOptExpLS` value written to the `localStorage`?

Reply:
It will be written to localStorage in Optimizely itself, it has a "Variation Code Editor" where you can add code that it will run based on the given variation before it fully loads the page.